### ISSUE TITLE
这个代码可以在jetson tx2 板子上面跑吗（Can this code run on the Jetson TX2 board？？）

### ISSUE THREAD

**Issue by Lianglmx** [qqwweee/keras-yolo3#718]

**Comment by robisen1** [comment]
Yes. While I dont use it now I have gotten it to run on a Nano and TX2 using various carrier boards. Note this code will run slow on the jetson unless you use tiny yolo. on Jetson, you could then expect anything from 7 to 12 FPS. there are other things you can do to push FPS up but that is too much for me to respond in a meaningful way.

**Comment by heshuju** [comment]
I have test, only tiny can run on jeston Nano.
the othor do not have enough memory(4G)
